src/attach-shadow.js

```diff
@@ -654,8 +655,8 @@ if (window['customElements'] && utils.settings.inUse && !utils.settings['preferP
   // NOTE: Instead of patching customElements.define,
   // re-define on the CustomElementRegistry.prototype.define
   // for Safari 10 compatibility (it's flakey otherwise).
-  Object.defineProperty(window['CustomElementRegistry'].prototype, 'define', {
-    value: function(name, constructor) {
+  //Object.defineProperty(window['CustomElementRegistry'].prototype, 'define', {
+  window.customElements.define = function(name, constructor) {
```

Comment:
If we're keeping this change, let's add a comment for why... did something else patch the instance rather than the prototype?

Reply:
Yeah, ShadyCSS patches this directly. Changing is path of least resistance.

Reply:
We could change this though, correct? https://github.com/webcomponents/shadycss/blob/f377017e50f4d472a57ac224973f88b63a92f17e/src/style-placeholder.js#L54

Reply:
The "fast" (tearoff) native shim also patches the instance.  So nevermind, let's just align on instance patching for now.